hardhat.config.js

```diff
@@ -1,125 +1,124 @@
-/**
- * @type import('hardhat/config').HardhatUserConfig
- */
 require("@nomiclabs/hardhat-waffle")
-require("@nomiclabs/hardhat-ethers")
-require("@nomiclabs/hardhat-truffle5")
 require("@nomiclabs/hardhat-etherscan")
 require("hardhat-deploy")
-require("hardhat-gas-reporter")
 require("solidity-coverage")
-require('hardhat-contract-sizer');
-require("./tasks/accounts")
-require("./tasks/balance")
-require("./tasks/withdraw-link")
-require("./tasks/block-number")
-require("./tasks/price-consumer")
-require("./tasks/random-number-consumer")
-require("./tasks/api-consumer")
-require("./tasks/keepers")
+require("hardhat-gas-reporter")
+require("hardhat-contract-sizer")
+require("./tasks")
 require("@appliedblockchain/chainlink-plugins-fund-link")
+require("dotenv").config()
 
-require('dotenv').config()
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
 
-const MAINNET_RPC_URL = process.env.MAINNET_RPC_URL || process.env.ALCHEMY_MAINNET_RPC_URL || "https://eth-mainnet.alchemyapi.io/v2/your-api-key"
-const RINKEBY_RPC_URL = process.env.RINKEBY_RPC_URL || "https://eth-rinkeby.alchemyapi.io/v2/your-api-key"
+const MAINNET_RPC_URL =
+  process.env.MAINNET_RPC_URL ||
+  process.env.ALCHEMY_MAINNET_RPC_URL ||
+  "https://eth-mainnet.alchemyapi.io/v2/your-api-key"
+const RINKEBY_RPC_URL =
+  process.env.RINKEBY_RPC_URL || "https://eth-rinkeby.alchemyapi.io/v2/your-api-key"
 const KOVAN_RPC_URL = process.env.KOVAN_RPC_URL || "https://eth-kovan.alchemyapi.io/v2/your-api-key"
-const POLYGON_MAINNET_RPC_URL = process.env.POLYGON_MAINNET_RPC_URL || "https://polygon-mainnet.alchemyapi.io/v2/your-api-key"
-const MNEMONIC = process.env.MNEMONIC || "your mnemonic"
-// optional
+const POLYGON_MAINNET_RPC_URL =
+  process.env.POLYGON_MAINNET_RPC_URL || "https://polygon-mainnet.alchemyapi.io/v2/your-api-key"
 const PRIVATE_KEY = process.env.PRIVATE_KEY || "your private key"
```

Comment:
If we want to close issue #74 we should 
```suggestion
const PRIVATE_KEY = process.env.PRIVATE_KEY
```

and then in each `networks` object make private key optional like this
```
accounts: PRIVATE_KEY !== undefined ? [PRIVATE_KEY] : []
```